msresist/figures/figureM3.py

```diff
@@ -53,19 +53,18 @@ def makeFigure():
     tt = centers.iloc[:, -1]
     tt = tt.replace("Normal", 0)
     tt = tt.replace("Tumor", 1)
-    lr = LogisticRegressionCV(cv=4, solver="saga", max_iter=10000, n_jobs=-1, penalty="elasticnet", class_weight="balanced", l1_ratios=[0.5, 0.9]).fit(c, tt)
+    svc = LinearSVC(penalty="l1", dual=False, max_iter=10000, tol=1e-7)
```

Comment:
You're still using LinearSVC here?

Reply:
Oops forgot to change this one. Solved below.